packages/wallet/jest/contract-test-setup.ts

```diff
@@ -0,0 +1,29 @@
+import "../config/env";
+import {GanacheServer} from "@statechannels/devtools";
+const {deploy} = require("../deployment/deploy");
+
+export default async function setup() {
+  const ganacheServer = new GanacheServer(
+    Number(process.env.GANACHE_PORT),
+    Number(process.env.CHAIN_NETWORK_ID)
+  );
+  await ganacheServer.ready();
+
+  const {
+    CONSENSUS_APP_ADDRESS,
+    TRIVIAL_APP_ADDRESS,
+    NITRO_ADJUDICATOR_ADDRESS,
+    ETH_ASSET_HOLDER_ADDRESS,
+    TEST_TOKEN_ADDRESS,
+    TEST_TOKEN_ASSET_HOLDER_ADDRESS
+  } = await deploy();
+
+  process.env.CONSENSUS_APP_ADDRESS = CONSENSUS_APP_ADDRESS;
```

Comment:
```suggestion
  process.env = { ...process.env, deployedArtifacts };
```
(paired with the above suggestion)